src/components/VSearchResultsTitle.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <h1
+    class="capitalize leading-tight sr-only md:not-sr-only"
```

Comment:
```suggestion
    class="uppercase leading-tight sr-only md:not-sr-only"
```

This is probably preferable.